appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -425,6 +426,7 @@ public MockForm(SimpleEditor editor) {
 
     // Initialize mock form UI by adding the phone bar and title bar.
     changePreview();
+    phoneBar = new PhoneBar();
```

Comment:
changePreview should set up the PhoneBar, so I'm not sure why this is needed here.